RELEASE/scripts/autoscend/auto_mr2018.ash

```diff
@@ -287,8 +287,8 @@ boolean fantasyRealmToken()
 		}
 	}
 
-	// If we're not allowed to adventure without a familiar
-	if(forbidFamChange() && auto_have_familiar($familiar[Mosquito]))
+	// If we're not allowed to adventure without a familiar due to being in a 100% familiar run.
+	if(is100FamRun())
```

Comment:
Nice. This is technically a bugfix for the paths that don't have Familiars as they'll be able to get Phat Loot Tokens from FantasyRealm again 👍

Reply:
Yep. this is exactly one of the things I wanted to fix but needed a function that actually answers "are we really in a 100% run, not just forbidden to change familiar"